Demo/Demo/Models/SetupTokenRequest.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import CorePayments
+
+struct SetUpTokenRequest {
+    
+    let customerID: String?
+    
+    var path: String {
+        return "/setup_tokens/"
```

Comment:
Same comments as above in `path`, `method`, and `headers` here we can rely on implicit returns

Reply:
03ecf9ae0cba712d182793350a8c69849136aac5